plugin/evm/orderbook/hubbleutils/margin_math.go

```diff
@@ -90,7 +100,7 @@ func GetPositionMetadata(price *big.Int, openNotional *big.Int, size *big.Int, m
 }
 
 func GetNotionalPosition(price *big.Int, size *big.Int) *big.Int {
-	return big.NewInt(0).Abs(Div1e18(Mul(size, price)))
+	return big.NewInt(0).Abs(Div1e18(Mul(price, size)))
```

Comment:
does the ordering make any difference, readability-wise?

Reply:
hehe just writing in the same order of given arguments